src/app/content/components/Page/PageNotFound.tsx

```diff
@@ -46,7 +46,7 @@ const PageNotFound = () => <PageNotFoundWrapper>
         {(msg) => msg}
       </FormattedMessage>
     </span>
-    <SidebarControl />
+    <OpenSidebarControl hideMobileText={false}/>
```

Comment:
@TomWoodward 
Do we want to always keep the text or make the sidebar control to behave and look same as in toolbar?